### DISCUSSION TITLE
Can tiles be changed on the 3ds version?

### DISCUSSION THREAD

**Discussion by ROXYAZURE** [angband/angband#5863] [Q&A] [answered]
I've seen the game includes a tiles folder, but I can't find a way to change them. Only Ascii mode seems to be available. 
Is there a way to change graphics in the 3ds port of the game?

**Comment by NickMcConnell** [comment]
Playing in windows?

**Reply by ROXYAZURE** [reply]
I play on a N3dsXL

**Comment by NickMcConnell** [comment]
Ah, I think you can't use tiles there, but I'm not 100% sure.  @timschumi?

**Reply by ROXYAZURE** [reply]
I do see the tiles folder in the game's folder.

**Reply by timschumi** [reply, marked as answer]
Tiles are currently unsupported, I ripped out support for them while overhauling the port. Not sure if they will ever make a comeback, all current tilesets are bigger than the 5x8 (3x8 for NDS respectively) font that we _barely_ manage to fit on screen.

They are really just there because I was too lazy to remove them from the packaged directory manually.

**Reply by ROXYAZURE** [reply]
Alright, thanks for the answer, I was curious as to why tiles wouldn't work if they are in the game files.
